docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss

```diff
@@ -52,6 +52,8 @@
 }
 
 @mixin hb-unordered-list {
+  display: flex;
```

Comment:
what does this do?

Reply:
The combination of `display: flex` used with `flex-direction: column` allow the bullets and numbers on unordered and ordered lists to align correctly when lists are used alongside floated images. Honestly, I was struggling to figure this one out and then thought, hmmm..... I wonder what flexbox would do and :boom: it worked! But I'm open to other ideas. 💡 

Here's a screenshot of before:

![before](https://user-images.githubusercontent.com/12678977/81016962-6eb23000-8e2f-11ea-9d43-f314ea956858.png)

And a screenshot of after:

![after](https://user-images.githubusercontent.com/12678977/81016971-72de4d80-8e2f-11ea-82a2-3ff2e5364c68.png)

Reply:
ok! that makes sense, I guess I am now  just wondering if we should only applyy display flex to lists when they are in a wysiwyg textarea, and not all the time. Maybe issues could arise if that's always applied.

Reply:
Yes!!! I removed the the flexbox styles from the mixins and added them to the WYSIWYG to narrow the scope. Good call!